os/tools/mkbinheader.py

```diff
@@ -94,20 +96,74 @@ def get_static_ram_size(bin_type):
             rosize = roundup_power_two(rosize)
             textsize = roundup_power_two(textsize)
         static_ram_size = textsize + rosize + datasize + bsssize
+        ram_fp.close()
+        os.remove(STATIC_RAM_ESTIMATION)
+        return static_ram_size
     else : #Not supported.
-        print("Error : Not supported Binary Type")
+        print("Error : Not supported Binary Format")
         sys.exit(1)
 
-    ram_fp.close()
-    os.remove(STATIC_RAM_ESTIMATION)
-    return static_ram_size
+############################################################################
+#
+# Kernel binary header information :
+#
+# total header size is (12 + #)bytes.
+# +-----------------------------------------------------------------------------------
+# | Header size | Binary Version |  Binary Size | Secure Header Size | Secure Header |
+# |   (2bytes)  |    (4byte)     |    (4byte)   |       (2byte)      |    (#bytes)   |
```

Comment:
Kernel version is YYMMDD format as you said in Kconfig.
Then do we need 4bytes?